test/test_audio.rb

```diff
@@ -50,68 +50,54 @@ def test_sample
     refute channel.paused?
     refute channel.playing?
     
-    channel = sound.play_pan(1, 1, 0.5, false)
+    channel = sound.play_pan(1, volume: 1, speed: 0.5, looping: false)
     interactive_cli("Right speaker again, but slowed down this time?")
     
   ensure
     channel && channel.stop
   end
-  
-  class SongTestWindow < Gosu::Window
-    def initialize
-      super(50, 50)
-      self.caption = "Song-Test"
-    end
-  end
-  
+
   def test_song
     skip_on_appveyor
 
-    win = SongTestWindow.new
-    
-    assert_nil Gosu::Song.current_song
+    # assert_nil Gosu::Sound.current_songs
     
-    song = Gosu::Song.new(media_path("0830.ogg"))
+    song = Gosu::Sound.new(media_path("0830.ogg"), streaming: true)
 
-    song.play(true)
+    channel = song.play(looping: true)
```

Comment:
Really not sure of this, but most folks already adapted the Sample.play = Channel logic, so why not make it the same for "Songs"